tests_utils/date_time_utils.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+class DateTimeUtils:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def current_datetime(format_date="%d/%m/%Y %H:%M"):
```

Comment:
Can I get you to specify the types on these methods?

So for this one, I think it'll be: `def current_datetime(format_date: str = "%d/%m/%Y %H:%M") -> str:`